cloud-data-protection/src/features/userSlice.tsx

```diff
@@ -0,0 +1,27 @@
+import {createSlice, PayloadAction} from '@reduxjs/toolkit';
+import {LoginResult} from "../services/result/loginResult";
+
+export const userSlice = createSlice({
+    name: 'user',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/olivierbouchomsfontys/cloud-data-protection/pullRequest?prid=7219402)
```suggestion
    name: "user",
```